Test/Framework/Visual/BlendStateTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using Microsoft.Xna.Framework.Graphics;
+using NUnit.Framework;
+
+namespace MonoGame.Tests.Visual
+{
+    [TestFixture]
+    internal class BlendStateTest : VisualTestFixtureBase
+    {
+        [Test]
+        public void ShouldNotBeAbleToSetNullBlendState()
+        {
+            Game.DrawWith += (sender, e) =>
+            {
+                Assert.Throws<ArgumentNullException>(() => Game.GraphicsDevice.BlendState = null);
+            };
+            Game.Run();
+        }
+
+        [Test]
+        public void ShouldNotBeAbleToMutateStateObjectAfterBindingToGraphicsDevice()
+        {
+            Game.DrawWith += (sender, e) =>
+            {
+                var blendState = new BlendState();
+                Assert.DoesNotThrow(() => blendState.AlphaBlendFunction = BlendFunction.ReverseSubtract);
```

Comment:
> I always go back and forth on this

I do some as well, but our goal is 100% code coverage... even if the test has a low likelihood of failing.